templates/quarkus/events/src/main/java/functions/Function.java

```diff
@@ -3,21 +3,15 @@
 import io.quarkus.funqy.Context;
 import io.quarkus.funqy.Funq;
 import io.quarkus.funqy.knative.events.CloudEvent;
+import io.quarkus.funqy.knative.events.CloudEventBuilder;
 
 public class Function {
 
     @Funq
-    public Output function(Input input, @Context CloudEvent cloudEvent) {
-        if (cloudEvent != null) {
-            System.out.println(
-                    "CloudEvent{" +
-                            "id='" + cloudEvent.id() + '\'' +
-                            ", specVersion='" + cloudEvent.specVersion() + '\'' +
-                            ", source='" + cloudEvent.source() + '\'' +
-                            ", subject='" + cloudEvent.subject() + '\'' +
-                            '}');
-        }
-        return new Output(input.getMessage());
+    public CloudEvent<Output> function(CloudEvent<Input> input) {
```

Comment:
Previous version had one advantage: it accepted also non-CE HTTP (in which case the `cloudEvent` parameter was `null`).

Reply:
But non-CE HTTP works fine in the HTTP templates, so...